t/app/controller/dashboard.t

```diff
@@ -230,21 +243,37 @@ FixMyStreet::override_config {
         $mech->get_ok('/dashboard?export=1');
         like $mech->res->header('Content-type'), qr'text/csv';
         $mech->content_contains('Report ID');
+        $mech->delete_header('Authorization');
+    };
+};
+
+FixMyStreet::override_config {
+    ALLOWED_COBRANDS => 'tester',
+    MAPIT_URL => 'http://mapit.uk/',
+}, sub {
+    subtest 'no body or export, 404' => sub {
+        $mech->get('/dashboard');
+        is $mech->status, '404', 'No parameters, 404';
+        $mech->get('/dashboard?export=1');
+        is $mech->status, '404', 'If no body, 404';
+        $mech->get("/dashboard?body=$body_id");
+        is $mech->status, '404', 'If no export, 404';
+    };
+
+    subtest 'body and export, okay' => sub {
+        $mech->get_ok("/dashboard?body=$body_id&export=1");
     };
 };
 
 sub test_table {
     my ($content, @expected) = @_;
     my $res = $categories->scrape( $mech->content );
-    my $i = 0;
+    my @actual;
     foreach my $row ( @{ $res->{rows} }[1 .. 11] ) {
-        foreach my $col ( @{ $row->{cols} } ) {
-            is $col, $expected[$i++];
-        }
+        push @actual, @{$row->{cols}} if $row->{cols};
     }
+    is_deeply \@actual, \@expected;
```

Comment:
I am being picky here but it might be nice if this unrelated change was in it's own commit, or explained in the commit message.